python/cudf/cudf/tests/test_setitem.py

```diff
@@ -45,6 +45,23 @@ def test_dataframe_setitem_existing_columns(df, arg, value):
     assert_eq(df, gdf, check_dtype=False)
 
 
+@pytest.mark.parametrize(
+    ("df", "arg", "value", "expected"),
+    [
+        (DataFrame(), ["a", "b"], 0, DataFrame({"a": [], "b": []})),
```

Comment:
To be clear, are we testing the following? If so, it feels wrong...

```python
df = DataFrame()
df[["a", "b"]] = 0
assert_eq(df, DataFrame({"a": [], "b": []})
```

Reply:
This was supported by earlier version of cudf, but pandas throws error with similar approach.

Reply:
It seems to me like it's a bug that we should no longer "support" :) Is there anywhere within cudf that we depend on this behaviour?

Reply:
No we don't, it was request from user in issue #4657